### DISCUSSION TITLE
configuring user/group permissions

### DISCUSSION THREAD

**Discussion by jronpaul** [requarks/wiki#3422] [Help / Questions]
**Question**
Hi,
im trying to isolate groups to specific pages..
which i have working however I can only log in via the direct path.. not the home page..
is there anyway to get users to be put in their designated area?
for example, if i have a user in a group that is only allowed to see /home/IT
i get denied if i try to log in as the user on the default page but if i do http://ip/home/IT
it works..

**Host Info (please complete the following information):**
Red Hat Enterprise Linux Server release 7.7 (Maipo)
Wiki.js 2.4.107
postgres12

**Comment by jianbin-wei** [comment]
I have the same issue.  

I suspect it is because the redirect does not work correctly.  If I logout and login again, it will redirect to the correct path.  However the redirect does not work consistently.  Also if a new page is created under different path then weird error happens.  

I played around with this feature as documented at https://docs.requarks.io/groups (Private Wiki and Departments) and had no luck yet.

**Comment by jronpaul** [comment]
me either... I cant seem to get it to work..
the permissions are working for me as far as keeping users in certain pages, but it does not bring them there for the initial login and if you click the home button, you get denied as well..
Does anyone in here acknowledge when there is a bug ? Not sure if i should post it anywhere else.

**Comment by jronpaul** [comment]
> 
> 
> I have the same issue.
> 
> I suspect it is because the redirect does not work correctly. If I logout and login again, it will redirect to the correct path. However the redirect does not work consistently. Also if a new page is created under different path then weird error happens.
> 
> I played around with this feature as documented at https://docs.requarks.io/groups (Private Wiki and Departments) and had no luck yet.

Jianbin,
i got this to work by setting another permission to allow read pages,upload assets, edit + delete assets, read comments, edit comments to my home page.

![perms](https://user-images.githubusercontent.com/62318524/92243527-9d005480-ee8f-11ea-9f22-0dcb5245e7d3.PNG)

**Comment by jianbin-wei** [comment]
Thanks a lot for this.  Will try 😀

**Comment by jpc54321** [comment]
I have noticed that if you use the 'Login as' button when you receive the 'Unauthorised' page the redirect works but, you end up with increased page permissions when these are not selected as part of the 'page rules'
